image/system_a/data/lang/English.json

```diff
@@ -372,6 +372,8 @@
     "app_phonebook_new_contact_email": "Email",
     "app_phonebook_new_contact_first_name": "First name",
     "app_phonebook_new_contact_invalid_number": "<text>Cannot save this contact.<br></br>The phone number you entered <br></br>is in an invalid format.</text>",
+    "app_phonebook_new_contact_unable_to_save": "Unable to save contact",
+    "app_phonebook_new_contact_same_numbers": "<text>Cannot save this contact.<br></br>The phone numbers you entered <br></br>are practically the same.</text>",
```

Comment:
IMO this copy doesn't sound right. Personally I'd be very confused seeing such message when trying to add a contact - what does it mean that phone numbers are "practically the same" and what is the value of presenting such message to user? The differences between phone numbers can be defined in a exact, measurable way - the condition that causes this message to appear is that in the phonebook the user already has a number that only differs by (not) having country code. That is the info I'd prefer to see as a user so I can understand why I'm not able to add the contact and what I shall do to make it possible.

Reply:
After discussions the text is changed already.